src/pages/RepoPage/BundlesTab/BundleOnboarding/BundleOnboarding.spec.tsx

```diff
@@ -0,0 +1,154 @@
+
+        const viteOnboarding = screen.getByText('ViteOnboarding')
+        expect(viteOnboarding).toBeInTheDocument()
+      })
+    })
+  })
+
+  describe('on /new/rollup route', () => {
+    describe('rendering tabs', () => {
+      it('renders selected vite tab', async () => {
+        render(<BundleOnboarding />, {
+          wrapper: wrapper('/gh/codecov/test-repo/bundles/new/rollup'),
+        })
+
+        const viteTab = await screen.findByText('Vite')
+        expect(viteTab).toBeInTheDocument()
+        expect(viteTab).not.toHaveAttribute('aria-current', 'page')
+      })
+
+      it('renders rollup tab', async () => {
+        render(<BundleOnboarding />, {
+          wrapper: wrapper('/gh/codecov/test-repo/bundles/new/rollup'),
+        })
+
+        const rollupTab = await screen.findByText('Rollup')
+        expect(rollupTab).toBeInTheDocument()
+        expect(rollupTab).toHaveAttribute('aria-current', 'page')
+      })
+
+      it('renders webpack tab', async () => {
+        render(<BundleOnboarding />, {
+          wrapper: wrapper('/gh/codecov/test-repo/bundles/new/rollup'),
+        })
+
+        const webpackTab = await screen.findByText('Webpack')
+        expect(webpackTab).toBeInTheDocument()
+        expect(webpackTab).not.toHaveAttribute('aria-current', 'page')
+      })
+    })
+
+    describe('rendering body', () => {
+      it('renders rollup onboarding', () => {
+        render(<BundleOnboarding />, {
+          wrapper: wrapper('/gh/codecov/test-repo/bundles/new/rollup'),
+        })
+
+        const rollupOnboarding = screen.queryByText('RollupOnboarding')
+        expect(rollupOnboarding).not.toBeInTheDocument()
+      })
+    })
+  })
+
+  describe('on /new/webpack route', () => {
+    describe('rendering tabs', () => {
+      it('renders selected vite tab', async () => {
+        render(<BundleOnboarding />, {
+          wrapper: wrapper('/gh/codecov/test-repo/bundles/new/webpack'),
+        })
+
+        const viteTab = await screen.findByText('Vite')
+        expect(viteTab).toBeInTheDocument()
+        expect(viteTab).not.toHaveAttribute('aria-current', 'page')
+      })
+
+      it('renders rollup tab', async () => {
+        render(<BundleOnboarding />, {
+          wrapper: wrapper('/gh/codecov/test-repo/bundles/new/webpack'),
+        })
+
+        const rollupTab = await screen.findByText('Rollup')
+        expect(rollupTab).toBeInTheDocument()
+        expect(rollupTab).not.toHaveAttribute('aria-current', 'page')
+      })
+
+      it('renders webpack tab', async () => {
+        render(<BundleOnboarding />, {
+          wrapper: wrapper('/gh/codecov/test-repo/bundles/new/webpack'),
+        })
+
+        const webpackTab = await screen.findByText('Webpack')
+        expect(webpackTab).toBeInTheDocument()
+        expect(webpackTab).toHaveAttribute('aria-current', 'page')
+      })
+    })
+
+    describe('rendering body', () => {
+      it('renders webpack onboarding', () => {
```

Comment:
Does this test render webpack onboarding? Looks like we are asserting that it does not

Reply:
It will in the future, the following PRs adjust this to be a truthful statement.